Neos.Flow/Tests/Unit/Persistence/Doctrine/PersistenceManagerTest.php

```diff
@@ -78,7 +77,19 @@ public function setUp()
         $this->mockSystemLogger = $this->createMock(LoggerInterface::class);
         $this->persistenceManager->injectLogger($this->mockSystemLogger);
 
-        $this->inject($this->persistenceManager, 'throwableStorage', $this->getMockBuilder(ThrowableStorageInterface::class)->getMock());
+        $mockThrowableStorage = $this->getMockBuilder(ThrowableStorageInterface::class)->getMock();
+        $this->inject($this->persistenceManager, 'throwableStorage', $mockThrowableStorage);
+
+        $allowedObjectsContainer = new AllowedObjectsContainer();
+        $this->inject($this->persistenceManager, 'allowedObjects', $allowedObjectsContainer);
+        $allowedObjectsListener = new AllowedObjectsListener();
```

Comment:
Not a mock, because we actually depend on the code in the listener being executed.